composer.json

```diff
@@ -15,7 +15,6 @@
         "drupal/config_devel": "~1.2",
         "drupal/drupal-extension": "~4.0",
         "drupal/tmgmt": "1.11",
-        "drush/drush": "~10.3",
```

Comment:
Sorry for not being explicit in my comment on the line of guzzle. I meant, guzzle is drupal core dependency. Revert this change.